scripts/fix_files.py

```diff
@@ -0,0 +1,153 @@
+import os
+import sys
+import io
+import mimetypes
+import tempfile
+import zipfile
+from typer import run, Option, Argument
+from concurrent.futures import FIRST_EXCEPTION
+from concurrent.futures import Future
+from concurrent.futures import ThreadPoolExecutor
+from concurrent.futures import wait
+from typing import NamedTuple
+import django
+from django.db import migrations, models
+import requests
+from requests_cache import CachedSession
+
+
+class _Result(NamedTuple):
+    dist: models.Model  # DatasetDistribution
+    format: str
+
+
+def main(cache_location: str = Argument(..., help="File name to cache requessts")):
+    formats = {
+        f.extension: f
+        for f in Format.objects.all()
+    }
+
+    client = CachedSession(cache_location)
```

Comment:
Would be nice to have a default location set, preferably using XDG_CACHE_HOME (~/.cache/vitrina/dist.db).

https://wiki.archlinux.org/title/XDG_Base_Directory